etc/scripts/generateDag.py

```diff
@@ -1,41 +1,42 @@
 #!/usr/bin/env python
 
-# 
+#
 # LSST Data Management System
 # Copyright 2008, 2009, 2010 LSST Corporation.
-# 
+#
 # This product includes software developed by the
 # LSST Project (http://www.lsst.org/).
 #
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by
 # the Free Software Foundation, either version 3 of the License, or
 # (at your option) any later version.
-# 
+#
 # This program is distributed in the hope that it will be useful,
 # but WITHOUT ANY WARRANTY; without even the implied warranty of
 # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 # GNU General Public License for more details.
-# 
-# You should have received a copy of the LSST License Statement and 
-# the GNU General Public License along with this program.  If not, 
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
 # see <http://www.lsstcorp.org/LegalNotices/>.
 #
 
-import math
+from __future__ import print_function
+from builtins import str
+
 import argparse
-import os
-import subprocess
+
 import sys
-import time
 
-from textwrap import dedent
-import glob
-import re
+try:
+    from shlex import split as cmd_split
+except ImportError:
+    from pipes import split as cmd_split
```

Comment:
I'm not sure what happened there.   Must have been a copy buffer paste.  Thanks for catching that.